### PR TITLE
D10 prep

### PR DESCRIPTION
**Pull request by attiks**

### CODE REVIEW

File: html/modules/disabled/formdazzle/formdazzle.info.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name: 'formdazzle - disabled'
```
Comment (lazysoundsystem) [inline html/modules/disabled/formdazzle/formdazzle.info.yml:1]: To import the config, I needed a disabled module for 'ckeditor' too.

**Review by lazysoundsystem — COMMENTED** [review]
Had some strange issues with my local setup which I think were unrelated to this change.

Not sure if those explain why I'm not getting any actual figures:
image
but everything else seems to be okay.

As noted, I needed a 'disabled' stub for 'ckeditor' module to be able to import the config.

File: composer.json [hunk 2]
```diff
@@ -34,23 +34,20 @@
...
         "drupal/csp": "^1.16",
         "drupal/environment_indicator": "^4.0",
         "drupal/flexible_permissions": "^1.0@beta",
-        "drupal/formdazzle": "^2.1",
         "drupal/google_tag": "^1.4",
         "drupal/group": "^2.0.0-rc1",
         "drupal/inline_entity_form": "^1.0@RC",
         "drupal/layout_paragraphs": "^2.0@beta",
         "drupal/link_allowed_hosts": "^1.0@beta",
-        "drupal/maintenance200": "^1.1",
```
[the remark this answers is not in the document]
Comment (attiks) [reply composer.json]: Added

File: html/modules/disabled/formdazzle/formdazzle.info.yml (hunk 1 above)
Comment (attiks) [reply html/modules/disabled/formdazzle/formdazzle.info.yml:1]: Added

**Review by lazysoundsystem — APPROVED** [review]
Thanks for the fixes - I'll look to see why I don't have figures showing, but imagine they will do on the dev server.

**Comment by attiks** [comment]
> Thanks for the fixes - I'll look to see why I don't have figures showing, but imagine they will do on the dev server.

You'll need settings

```
$config['ocha_key_figures.settings']['ocha_api_url'] = 'https://xxx:xxx@dev.api-unocha-org.ahconu.org/api/v1/';
$config['ocha_key_figures.settings']['ocha_api_key'] = 'secret';
```

**Pull request merged**